charts/csm-replication/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v2
+name: csm-replication
+type: application
+version: 1.3.0
+appVersion: "1.3.0"
```

Comment:
Latest version of csm-replication is 1.3.1
Please also update the same in values.yaml file.

Reply:
Thanks, Santhosh, for revieing the code, I have updated the version in the required files.

Reply:
Only the dell-replication-controller is at v1.3.1, not the dell-csi-replicator sidecar whose version still remains at v.1.3.0.
Please revert those.

Reply:
Thanks Santhosh, Reverted the changes as mentioned. Please check and let me know if any changes need to be done.